### ISSUE TITLE
Wayland socket does not work

### ISSUE THREAD

**Issue by 239** [flathub/io.kopia.KopiaUI#34]
Fails to start after the last update (#33):
```
[2:0911/135659.281061:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:0911/135659.559466:ERROR:ozone_platform_x11.cc(240)] Missing X server or $DISPLAY
[2:0911/135659.559480:ERROR:env.cc(255)] The platform failed to initialize.  Exiting.
The futex facility returned an unexpected error code.
```

**Comment by darkdragon-001** [comment]
Thanks for reporting and sorry for breaking. Should be fixed in #36.

**Comment by 239** [comment]
Yes, works again - thanks for the quick fix!
